Trivia/plugin.py

```diff
@@ -214,11 +214,30 @@ def event():
             if self.active:
                 schedule.addEvent(event, eventTime, 'next_%s' % self.channel)
 
-
+        def skip(self):
+          self.reply(_("Question skipped. The answer was: %s" % self.a[0]))
```

Comment:
No, it should be: `self.reply(_("Question skipped. The answer was: %s") % self.a[0])`.

The reason is that _() returns a translation of the template instead of the final string. (Templates can be translated ahead of time)